app/publishing/delinkedCalculation/update-published.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../../data')
+
+const updatePublished = async (calculationReference, transaction) => {
+  await db.delinkedCalculation.update(
+    { datePublished: new Date() },
+    {
+      where: {
+        calculationId: calculationReference
```

Comment:
remember your variables can be whatever you want them to be - even if we stick with the remapped calculationReference, this function doesn't know that and just knows its been given a value. So no reason we can't recode this as the far neater:

`const db = require('../../data')

const updatePublished = async (calculationId, transaction) => {
  await db.delinkedCalculation.update(
    { datePublished: new Date() },
    {
      where: {
        calculationId
      },
      transaction
    }
  )
}

module.exports = updatePublished`

Reply:
Nice job mapping this as code, GH